### PR TITLE
#147 monthly task

### PR DESCRIPTION
**Pull request by prathameshzarkar9**
used tag 1-1.22-bullseye which pointing to the latest release for go

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {
 	"name": "Go",
 	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
-	"image": "mcr.microsoft.com/devcontainers/go:1-1.21-bullseye",
+	"image": "mcr.microsoft.com/devcontainers/go:1-1.22-bullseye",
```
Comment (samruddhikhandale) [inline .devcontainer/devcontainer.json:6]:
```suggestion
	"image": "mcr.microsoft.com/devcontainers/go:1-1.22-bookworm",
```

Can you test if dev container is created fine and steps in README work?

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (prathameshzarkar9) [reply .devcontainer/devcontainer.json:6]: tested the new image tag with all the `things to try`. it is working fine.
shall I change this to bookworm then?

Comment (samruddhikhandale) [reply .devcontainer/devcontainer.json]: > shall I change this to bookworm then?

Yep!

**Comment by prathameshzarkar9** [comment]
Hi @samruddhikhandale ,
Can you help review this PR and approve the same?

**Review by samruddhikhandale — APPROVED** [review]

**Comment by samruddhikhandale** [comment]
@bamurtaugh would you be able to help approve as part of Community PR Approvals? Thanks!

**Review by bamurtaugh — APPROVED** [review]

**Review by bhavyaus — APPROVED** [review]

**Pull request merged**